react/components/EditorContainer/Sidebar/ConfigurationList/ContentEditor/ConditionControls/Separator.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+const Separator: React.FunctionComponent = () => <div className="mv7" />
```

Comment:
Does it make sense to use `hr` instead of div in this case?

Reply:
I wasn't sure, so I checked the [\<hr> MDN page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr).

> The HTML \<hr> element represents a thematic break between paragraph-level elements: for example, a change of scene in a story, or a shift of topic within a section.

I don't think it applies to this case.